frontend/src/components/Budget/Transaction/TransactionList.tsx

```diff
@@ -14,9 +15,37 @@ interface Props {
   categories: Category[];
 }
 
-class TransactionList extends React.Component<Props> {
+interface State {
+  month: string;
+  description: string;
+  transactionType: string;
```

Comment:
'withdraw' | 'deposit' would be a more specific type for this.

Reply:
Changed.